scalableminds/webknossos
# Allow deleting datasets on disk

Adds new datastore route `DELETE      /datasets/:organizationName/:dataSetName/deleteOnDisk` (use with care)
Adds a tab to dataset edit view, where this route can be triggered

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- backend: test route `DELETE      /datasets/:organizationName/:dataSetName/deleteOnDisk` (with admin token)
- frontend: check out new view, hit the button, dataset should be moved to .trash directory within organization datasets directory. Should be one after refresh

### Open Questions:
 - how to hide this for non-admins? (how to pass user to view as prop?)
 - how to trigger dataset list refresh on dashboard after redirect

### Issues:
- fixes #4628 

------
